### DISCUSSION TITLE
Command Line Params for Innosetup instead of INF?

### DISCUSSION THREAD

**Discussion by byjrack** [git-for-windows/git#4423] [Q&A] [answered]
So been working with our internal team for packaging up Git and possibly using winget to keep things up to date.

One item that came up was if we could set CURLOption to WinSSL as part of that work since it's required in our space.  If I saveinf and then loadinf things seem to work as expected.  I was trying to use winget though and stick with command line args.  Taking the [CURLOption=WinSSL and dumping on the command line](https://github.com/git-for-windows/git/wiki/Silent-or-Unattended-Installation) doesn't seem to cut it.  

I can see in the installer.iss the wizard prompts, but given it's not a component it seems like I can't set it from the commandline unless the setup is compiled with some parameter parsing in the declarations.  Am I missing something?

**Comment by dscho** [comment, marked as answer]
You can specify the CURL option like this: `/o:CURLOption=WinSSL`. For more details, see https://github.com/git-for-windows/build-extra/commit/8764c72d83cd05a709fcb21700e42081d226037b.

**Reply by byjrack** [reply]
I must have read through that code like 3 times looking for an arg parser and completely missed it. 😳
